Detector/DetStudies/tests/options/SiWAnalysis_batch.py

```diff
@@ -0,0 +1,96 @@
+from Gaudi.Configuration import *
+
```

Comment:
This should be replaced with a simple example.
Please use relative paths and for the necessary input you may use arguments to that script (so you can still run it for larger scale production).